CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - TypeScript: `Argument` is no longer generic ([#16](https://github.com/cucumber/cucumber-expressions/pull/16))
 
+### Fixed
```

Comment:
Why "Fixed"?

As this not really part of the library, we may move these entry outside of the headings? Or "Added"?

Reply:
I haven't made any changes to the library itself, so I didn't want to add a section to `Added`, which would be a signal to bump the minor version.

But since we already have a `Changed` entry I think we may have to bump the major version, so I can move it to `Added`.

However, for future changes to the Try UI, I think we should maybe omit the changes from the changelog. The changelog is for the library, not the UI.

So I'll remove it :-)